extensions/ql-vscode/src/model-editor/modeling-store.ts

```diff
@@ -211,8 +211,8 @@ export class ModelingStore extends DisposableObject {
         ...methods,
         // Keep all methods that are already modeled in some form in the state
         ...Object.fromEntries(
-          Object.entries(state.modeledMethods).filter(([_, value]) =>
```

Comment:
Ideally this shouldn't have generated a linting error. I feel like this array arg where the first element is `_` is morally the same as two parameters where the first is `_`. But the rule unfortunately triggered here so I rewrote this to get around it.

We could instead do
```typescript
const value = entry[1];
```
if you feel that would be clearer.

Reply:
I think the reason that this is giving an error is because the `_` is unnecessary: if you this to `.filter(([, value]) =>`, it should still work.

Reply:
Oh interesting, I didn't know you could do that and not specify a name for the first element at all. I'll try that.

Although, there's a chance that syntax is also a little unclear because it's only a single extra comma, and people may not have seen this before.